ui/src/app/app.module.ts

```diff
@@ -76,6 +74,8 @@ export function getConfiguration(signInService: SignInService): Configuration {
     WizardSelectComponent,
     WizardModalComponent,
     WizardTreeParentComponent,
+    WizardTreeChildrenComponent,
+    WizardCriteriaGroupComponent,
```

Comment:
Another thing I hate thinking about is where to put things. Also boring. Any objections to me following this up with a PR to alphabetize this list so we don't have that problem?

Reply:
Fine by me.